docs/reference/api-overview.md

```diff
@@ -65,9 +65,13 @@ Currently, there are several API groups in use:
 *  The named groups are at REST path `/apis/$GROUP_NAME/$VERSION`, and use `apiVersion: $GROUP_NAME/$VERSION`
    (for example, `apiVersion: batch/v1`).  Full list of supported API groups can be seen in [Kubernetes API reference](/docs/reference/).
 
-There is a supported path to extending the API:
-* [Third Party Resources](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/extending-api.md)
-   are for users with very basic CRUD needs.
+There are two supported paths to extending the API with [custom resources](/docs/concepts/extensions/custom-resources/):
```

Comment:
@abiogenesis-now Yes! The plan was to split then rewrite both of them, but I never got around to the second step. LOL If you want to take that task on, we should talk about it. The concept piece should be more high level and talk about the API in terms of the Kubernetes architecture; whereas, the reference piece should more specific to using/accessing the API.